### DISCUSSION TITLE
About Thermodynamics of Information

### DISCUSSION THREAD

**Discussion by karel950505** [josePereiro/TankeFactory#14] [Discussions]
How can one concile the Second Law of Thermodynamic  with the erasure of a bit of information in a computational system ?

**Comment by josePereiro** [comment]
From [here](https://doi.org/10.3390/e26030203)

> Since the inception of thermodynamics, a delicate tension between physics and information has been unfolding. On the one hand, it is generally believed that knowledge of a system’s evolution will not, by itself, change that evolution. Simultaneously, what an observer can do with a system (i.e., extract work or decrease entropy) does depend upon the knowledge they possess. Since the Second Law of thermodynamics, roughly speaking, requires that the thermodynamic entropy of a closed system can only increase, a paradox emerges: can an intelligent being circumvent the laws of thermodynamics?

**Reply by josePereiro** [reply]
[Here my current knowledge of the subject] this is how I see it, for the second law to hold true, any gain in free energy in a system where no 'traditional' heat/work is applied, must be accompanied by an increase of  entropy of the surroundings. A Maxwell’s Demon, and similar Szilard’s engine, models one of such cases. It is Landauer’s principle which states how such entropy increases, by associating an energy cost to the erasing of a bit. 

So, the second law is saved because demons engines (given that all memories are finite) must erase its 'memories' to close the cycle, employing an amount of energy at least sufficient for the entropy of the whole system to not decrease. 

You can also check [this one](https://www.nature.com/articles/nphys1821)... It is an experimental example of information-to-energy conversion.

**Reply by josePereiro** [reply]
Of course, this rises more questions, like: Is it reading free? Why?

**Reply by ta-rekt** [reply]
Hey @karel950505 !

What you said @josePereiro is explained in detail in [this article](https://valbuena.fis.ucm.es/gisc/papers/nphys3230.pdf), page 6, the section called "Cost of measurement". It is a great review, not very long and quite accessible.
